docs/pages/access-controls/teleport-policy/getting-started-policy.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Teleport Policy
+description: A reference for Access Graph with Teleport Policy.
+---
+
+Teleport Policy streamlines and centralizes access management across your entire infrastructure.
+
+Teleport Policy with Access Graph provides a visual representation of the relationships between
```

Comment:
To me, "Teleport Policy with Access Graph" sounds like Access Graph is an add-on to Teleport Policy.

Reply:
Open to suggestions. Since we have a mismatch due to no 'policy' labeling whatsoever in the UI, we're trying to use both terms.